### DISCUSSION TITLE
ATTINY816 AREF Pin

### DISCUSSION THREAD

**Discussion by metehoca** [SpenceKonde/megaTinyCore#1174] [Q&A] [answered]
On the ATmega328P, when using the internal 1V1 analog reference, we can read the generated 1.1V from the AREF pin. We can even connect a decoupling capacitor between this pin and GND to stabilize the voltage. However, on the ATtiny816, I cannot read this voltage. Is there a way or a need to do so?

**Comment by hmeijdam** [comment, marked as answer]
You cannot. 
The architecture from the 328P is this.
You can see that the AREF pin is connected to the output of the internal reference and directly to the input of the 10 bit DAC
![image](https://github.com/user-attachments/assets/93e6eb17-5dc2-41f2-981c-ec2a31204ee8)

The architecture of the ADC in the 816 is totally different
VRAFA is an analog input and I see no connection from the output of the internal reference to that pin. Vref goes directly in the ADC

![image](https://github.com/user-attachments/assets/42e49b47-5835-48a4-912d-f215e53423cf)
